contents/blog/company-survey-h1-2024.md

```diff
@@ -0,0 +1,99 @@
+
+### Qualitative questions
+
+**What motivates you right now?** The most common themes people highlighted this time were the same as last.
+
+* **Team.** People used phrases such as: the team, great colleagues, helping the team, the great folks at PostHog at all levels, and improving things for teammates.
+
+* **Growth.** People used phrases such as: growing the company, seeing revenue going up, business growing, and numbers going in the right direction.
+
+* **Opportunities.** People used phrases such as: many learning opportunities, ability to be heard and try new things, so many opportunities lie ahead of us, and building a great product.
+
+**If you were to leave PostHog, what would be the reason?** There were a variety of answers to this question, all of them hypothetical, but all to be taken seriously.
+
+* **Career growth.** People used phrases such as: start my own company, getting back into management, a better opportunity for my career, a career growth move, and starting my own startup.
+
+* **The work.** People used phrases such as: not feeling like I'm making an impact, getting overwhelmed with non-fun work, stuck too long on challenging, repeating things, and burnout.
+
+* **Company stops growing.** People used phrases such as: business growth stalling, growth stalling or the company failing, and the company isn't growing anymore.
+
+### Highlights from comments
+
+These specific but anonymous quotes are broadly reflective of responses from multiple people. We’ve split them into two categories, positive and constructive, as they cover a wide range of topics.
+
+Positive:
+
+* Bouncing wacky ideas off of peeps, then building them at warp speed
+* Everyone's motivation to grow and see the company succeed.
+* Shipping a product people love and use
+* I feel bad for people who don't get to work somewhere like PostHog
+* I would say PostHog is a place where people who can work independently and without much process and support do best, but for me this 'light-touch' process works
+* All-hands is great!
+* I really love the weekly context bomb at the all hands
+
+Constructive:
+
+* I think the lack of general process and over-engineered ownership that is typically a big plus has gotten in the way of some products actually making the strides forward that they could have.
+* For the company as a whole, I think things are pretty good, but I do think that satisfaction is heavily dependent on the team you're on.
+* Our setup means we are always on the edge of exhaustion because we purposefully keep things lean and tight on numbers.
+* All ingredients are there but growing pains are apparent specifically around team management
+
+### What we’ll do next
+
+Our exec team has already gone through all the results and we’ve decided on a bunch of actions that are already in progress with a plan and owner to address them. We’ll run the next survey at the start of Q4 2024 using the same format.
```

Comment:
We can flesh out more:
- Our executive team reads all responses in full
- Charles is responsible for making sure that we identify an actions that might be needed - typically the follow up occurs over Slack or in a 1-1. A good way to follow up is to ask directly 'in the latest team survey you said X, could you tell me more?' Usually it is the relevant exec team member who follows up here. 
- It is worth noting that not every comment needs a follow up - the priority here is to collect information and get a sense of overall trends. The survey becomes a failure if it is used as the primary way for people to share feedback with PostHog (which we encourage through more transparent channels). This is partly why we only do it every 6 months.